src/js/dataMapping/contracts/additionalDetails.js

```diff
@@ -43,12 +43,12 @@ export const parentFields = [
     {
         label: 'IDC Type',
         field: '__special',
-        parse: () => null
+        parse: (data) => data.latest_transaction.contract_data.type_of_idc
```

Comment:
Unless you need to do custom parsing, you can just replace the `field` value with the field name and remove the `parse` function. These are mostly here to return `null` values before we had these fields in the API.